presto-main/src/main/java/com/facebook/presto/connector/system/CatalogSystemTable.java

```diff
@@ -67,8 +68,8 @@ public ConnectorTableMetadata getTableMetadata()
     public RecordCursor cursor(ConnectorTransactionHandle transactionHandle, ConnectorSession session, TupleDomain<Integer> constraint)
     {
         Builder table = InMemoryRecordSet.builder(CATALOG_TABLE);
-        for (Map.Entry<String, ConnectorId> entry : metadata.getCatalogNames().entrySet()) {
-            table.addRow(entry.getKey(), entry.getValue().toString());
+        for (Map.Entry<String, ConnectorId> entry : metadata.getCatalogNames(Optional.of(session.getIdentity())).entrySet()) {
+            table.addRow(entry.getKey(), entry.getValue());
```

Comment:
Oh, I think I missed that part during rebasing at `ConnectorId` type change from `String` before